interface-definitions/interfaces-tunnel.xml.in

```diff
@@ -239,7 +239,7 @@
                   </leafNode>
                   #include <include/interface/interface-parameters-key.xml.i>
                   #include <include/interface/interface-parameters-tos.xml.i>
-                  #include <include/interface/interface-parameters-ttl.xml.i>
+                  #include <include/interface/interface-tunnel-parameters-ttl.xml.i>
```

Comment:
Plese do not create a 95% copy of the tzl include. It is supported that defaultValues can be overridden by simply redefining the defaultValue node.

Check the PPPoE interface definition how the defaultValue for MTU is altered.

Reply:
Nice feature. Thanks.